### DISCUSSION TITLE
Add file type in evidences

### DISCUSSION THREAD

**Discussion by 0xN0x** [dfir-iris/iris-web#51] [Ideas]
Add a field for file type and use the `file` command to process it automatically.
It's common to have a file with the wrong extension and could be useful.

**Comment by whikernel** [comment]
Hi @0xN0x  

The files in evidences are actually processed client-side in JS, nothing is upload to the server. While we could implement a sort of `file` in JS by reading headers, it is not the purpose of the evidences feature. We would recommend doing it manually and then specify it in the description if it doesn't match. 

We'll convert this issue as a discussion to see if more people want the field tho. 

Thanks for the sharing !

**Reply by Zeik0s** [reply]
Then what is the quick action button "upload Evidence" for? This highly suggests something is transferred from the client to server.

**Reply by whikernel** [reply]
Indeed it's a bit misleading, that's why we added the red text indicating that the data is not uploaded to the server in the modal. 
This is only meant to help process the hash, size and filename locally. 
We will change the quick action text, it's a bit legacy.
